jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsSlaveTemplate/config.jelly

```diff
@@ -29,11 +29,11 @@
       <f:radioBlock inline="true" name="jclouds.useHardwareId" value="false"
                     title="Specify RAM and Cores" checked="${instance.hardwareId == ''}">
         <f:entry title="Min. RAM (MB)" field="ram">
-          <f:textbox default="512"/>
+          <f:number clazz="positive-number" min="1" step="1" default="512"/>
         </f:entry>
         
         <f:entry title="Min. No.of Cores" field="cores">
-          <f:textbox default="1"/>
+          <f:number clazz="positive-number" min="1.0" step="1.0" default="1.0"/>
```

Comment:
I used decimals because the `cores` varaiable is defined as a `double` in [JCloudsSlaveTemplate.java](https://github.com/jenkinsci/jclouds-plugin/blob/20c6ca884abb172b27d0af82545c8915ce08f618/jclouds-plugin/src/main/java/jenkins/plugins/jclouds/compute/JCloudsSlaveTemplate.java#L85).

It looks like jclouds intends this value to be `double` [here](https://github.com/jclouds/jclouds/blob/0339cd2fd07ecdcc42fa0ebc74f6c1d9014eebfd/compute/src/main/java/org/jclouds/compute/domain/internal/TemplateBuilderImpl.java#L120).